content/en/internal/configuration-database/_index.md

```diff
@@ -9,9 +9,36 @@ type: "docs"
 in progress, more documentation is currently available at
 [EGI's wiki](https://wiki.egi.eu/wiki/GOCDB). {{% /pageinfo %}}
 
-The Grid Configuration Database (also know as the GOCDB) contains general
+The EGI Configuration Database (previously known as the GOCDB) contains general
 information about the sites participating to the production infrastructure.
 
+It's a central registry to record information about the topology of the EGI
+e-Infrastructure. This includes entities such as Operations Centres, Resource
+Centres, service endpoints and their downtimes, contact information and roles of
+users responsible for operations at different levels.
+
 It's used by all the actors (end-users, site managers, NGI mangers, support
 teams, VO managers), by other tools and by third party middleware to discover
 information about the infrastructure topology.
+
+## Identity card
+
+<!-- markdownlint-disable line-length no-bare-urls -->
+
+| Property                     | Value                                                                 |
+| ---------------------------- | --------------------------------------------------------------------- |
+| Name                         | Configuration Database                                                |
+| Description                  | Central registry of the infrastructure topology                       |
+| URL                          | https://goc.egi.eu                                                    |
+| Support Email                | gocdb-admin@mailman.egi.eu                                            |
+| Helpdesk Support Unit        | EGI Services and Service components -> Configuration Database (GOCDB) |
+| Configuration Database entry | https://goc.egi.eu/portal/index.php?Page_Type=Site&id=335             |
+| Supplier                     | [UKRI](https://www.ukri.org/)                                         |
+| Roadmap                      | [Roadmap](https://wiki.egi.eu/wiki/GOCDB/Roadmap)                     |
+| Release notes                | [Release notes](https://github.com/GOCDB/gocdb/releases)              |
+| Issue tracker for developers | https://github.com/GOCDB/gocdb/issues                                 |
+| Source code                  | https://github.com/GOCDB                                              |
+| License                      | gLite/Apache 2                                                        |
+| Privacy Policy               | https://wiki.egi.eu/wiki/GOCDB/data_privacy                           |
```

Comment:
What about this? https://goc.egi.eu/privacy.html

Reply:
Updated